tests/test_butlerUtils.py

```diff
@@ -110,7 +111,7 @@ def setUp(self):
         self.assertIsInstance(self.dataCoordMinimal, dafButler.dimensions.DataCoordinate)
         # NB the type check below is currently using a non-public API, but
         # at present there isn't a good alternative
-        viewType = dafButler.core.dimensions._coordinate._DataCoordinateFullView
+        viewType = dafButler.dimensions._coordinate._DataCoordinateFullView
```

Comment:
Better to drop this `assertIsInstance` check or replace it with a check on something that is public, like `lsst.daf.butler.NamedKeyMapping`.  Leaving this dependency on a private symbol is just asking for you code to break unexpectedly in the future, and it's just a test, not a production use case.

Reply:
Done, thanks.